projects/angular/src/wizard/_wizard.clarity.scss

```diff
@@ -439,6 +444,9 @@
       text-align: left;
       text-transform: none;
       margin: 0;
+      height: auto;
+      min-height: variables.$clr_baselineRem_1_5;
```

Comment:
Could we use core token about that? It will be easy to backport too.

Reply:
I can update the v17 PR to use the cds token directly.